programs/server/config.xml

```diff
@@ -77,6 +77,9 @@
             <!-- NuRaft heart beat interval in millisecond, default is 500. -->
             <!-- <heart_beat_interval_ms>500</heart_beat_interval_ms> -->
 
+            <!-- NuRaft request operation timeout in millisecond, default is 3000. -->
```

Comment:
NuRaft log replocation timeout in millisecond, default is same with operation_timeout.